.github/workflows/release.yml

```diff
@@ -137,6 +137,8 @@ jobs:
   publish:
     needs: [linux, appimage, macOS, windows]
     runs-on: ubuntu-20.04
+    permissions:
+      contents: write
```

Comment:
What does `contents` here refer to / why is it needed?

Reply:
https://docs.github.com/en/rest/reference/permissions-required-for-github-apps#permission-on-contents gives the details of what are allowed.

I switched all of the neovim org's actions to read-only access by default, so we now need to grant specific privileges for the default token (or create our own).

Reply:
> I switched all of the neovim org's actions to read-only access by default

Where/how?

> we now need to grant specific privileges for the default token (or create our own).

Isn't the default for contents "read" anyway (from the link I've posted above)?

Reply:
> Where/how?

https://github.com/neovim/neovim/settings/actions

> Isn't the default for contents "read" anyway (from the link I've posted above)?

Nope, almost all scopes are `read/write` by default:

https://docs.github.com/en/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token

And important here to note:

_When the `permissions` key is used, all unspecified permissions are set to no access, with the exception of the metadata scope, which always gets read access._